cypress/e2e/tests/pages/explorer/workloads/pods.spec.ts

```diff
@@ -1,93 +1,249 @@
 import { WorkloadsPodsListPagePo, WorkLoadsPodDetailsPagePo } from '@/cypress/e2e/po/pages/explorer/workloads-pods.po';
 import { createPodBlueprint, clonePodBlueprint } from '@/cypress/e2e/blueprints/explorer/workload-pods';
 import PodPo from '@/cypress/e2e/po/components/workloads/pod.po';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import { generatePodsDataSmall } from '@/cypress/e2e/blueprints/explorer/workloads/pods/pods-get';
 
-describe('Cluster Explorer', () => {
-  beforeEach(() => {
+describe('Pods', { testIsolation: 'off', tags: ['@explorer', '@adminUser'] }, () => {
+  const workloadsPodPage = new WorkloadsPodsListPagePo('local');
+  const nsName = `namespace${ +new Date() }`;
```

Comment:
one for another pr, this should use the generic function that creates resource names given the test run

Reply:
good call. will address in the next pr